.ci/E2E-tests/execute.sh

```
@@ -24,17 +25,26 @@ if [ "$CONFIGURATION" = "mysql" ]; then
 export HOST_HOSTNAME="nginx"
 
+# Export test paths for docker compose to pick up
+if [ -n "$TEST_PATHS" ]; then
+    export PLAYWRIGHT_TEST_PATHS="$TEST_PATHS"
+    echo "Running filtered tests: $TEST_PATHS"
+else
+    export PLAYWRIGHT_TEST_PATHS=""
+    echo "Running all tests"
+fi
+
 cd docker
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)